addons/base_import_module/static/src/base_import_list_view.js

```diff
@@ -0,0 +1,13 @@
+/** @odoo-module */
+
+import { registry } from "@web/core/registry";
+import { listView } from "@web/views/list/list_view";
+import { ImportListRenderer } from "./base_import_list_renderer";
+
+
+export const importModuleListView = {
+    ...listView,
+    Renderer: ImportListRenderer,
```

Comment:
also rename here ImportModuleListRenderer (it is consistent with 'importModuleListView')